workflows/iuf/operations/s3-upload/s3-upload-template.yaml

```diff
@@ -136,3 +156,23 @@ spec:
       image: artifactory.algol60.net/csm-docker/stable/docker.io/alpine/git:2.32.0
       command: [sh, -c]
       args: ["DIFF_TIME=$(expr {{inputs.parameters.opend}} - {{inputs.parameters.opstart}}); echo $DIFF_TIME; echo $DIFF_TIME > /tmp/diff_time.txt"]
+  
+  - name: make-product-catalog-content
+    inputs:
+      parameters:
+      - name: global_params
+    outputs:
+      parameters:
+      - name: product-content
+        valueFrom:
+          path: /tmp/product-content
+    script:
+      image: registry.local/artifactory.algol60.net/csm-docker/stable/docker.io/iuf:v0.1.0
+      command: [bash]
+      source: |
+        ARRAY=$(echo '{{inputs.parameters.global_params}}' | jq -r '[.product_manifest.current_product.manifest.content.s3[] | {"bucket": .bucket, "key": .key}]')
+        if [ $? -ne 0 ]; then
+          ARRAY="[]"
+          echo "No S3 content found."
```

Comment:
One thing I happened to notice here, I think it probably makes sense to echo this to `stderr`.